tests/common/config_reload.py

```diff
@@ -61,10 +62,46 @@ def config_force_option_supported(duthost):
     return False
 
 
+def config_reload_with_minigraph_override(sonic_host, wait=120, start_bgp=True,
```

Comment:
>config_reload_with_minigraph_override [](http://example.com/codeflow?start=4&length=37)

The name is confusing. The purpose of next function `config_reload` is to config override and reload. The purpose of this function is to render a config file which will be used by next function.